.travis.yml

```diff
@@ -2,13 +2,29 @@
 language: node_js
 node_js:
   - "0.12"
-  - "iojs"
 
 sudo: false
 
 cache:
   directories:
     - node_modules
+env:
+  matrix:
+  - EMBER_TRY_SCENARIO=ember-1.10
+  - EMBER_TRY_SCENARIO=ember-1.11
+  - EMBER_TRY_SCENARIO=ember-1.12
+  - EMBER_TRY_SCENARIO=ember-release
+  - EMBER_TRY_SCENARIO=ember-beta
+  - EMBER_TRY_SCENARIO=ember-canary
+
+matrix:
+  fast_finish: true
+  allow_failures:
+    - env: EMBER_TRY_SCENARIO=ember-1.11
+    - env: EMBER_TRY_SCENARIO=ember-1.12
+    - env: EMBER_TRY_SCENARIO=ember-release
+    - env: EMBER_TRY_SCENARIO=ember-beta
+    - env: EMBER_TRY_SCENARIO=ember-canary
```

Comment:
As we get tests passing for various framework versions, we can disallow failures on those versions